cli/src/migrate.js

```diff
@@ -170,23 +170,37 @@ function teardown() {
 function validateMigration() {
   // check that `${project}_internal` exists
   const project = this.options.project_name;
+  const niceErr = (e) => {
```

Comment:
pretty sure you can just use `err.msg` if it's a reql error if you want to exclude the stack trace.